default.env

```diff
@@ -88,3 +88,15 @@ CHECKPOINT_SYNC_URL=
 
 # Set to anything other than empty to start the lighthouse included slasher.
 START_SLASHER=
+
+# Set to anything other than empty to set the suggested fee recipient via a `fee_recipient.txt` file placed in `fee_recipient/`.
+# The file should contain key value pairs corresponding to validator public keys and their associated fee recipient, see example_fee_recipient.txt.
```

Comment:
Where can I see this example_fee_recipient.txt file?

Reply:
We removed support for this feature because it was deemed an unnecessary maintenance burden (see https://github.com/sigp/lighthouse/issues/3264). The main ways to configure a fee recipient now are using the flag, or the value in the `validator_definitions.yml` (which you could add without any extra tooling in `lighthouse-docker`).

Reply:
Docs here: https://lighthouse-book.sigmaprime.io/suggested-fee-recipient.html